examples/jib/README.adoc

```diff
@@ -0,0 +1,31 @@
+=== Example: Jib
+:icons: font
+
+Jib is one of the supported builders in Skaffold.
+[Jib](https://github.com/GoogleContainerTools/jib) builds Docker and OCI images
+for your Java applications and is available as plugins for Maven and Gradle.
+
+The way you configure it in `skaffold.yaml` is the following build stanza:
```

Comment:
I think it's worth mentioning that this example is for a standalone Maven project, where all dependencies are resolved from outside. The Jib builder requires that the projects are configured to use the Jib plugins for Maven or Gradle.  Multi-module builds require a bit additional configuration.